docs/design/datamover.md

```diff
@@ -214,25 +200,15 @@ spec:
     copyMethod: None
 ```
 
-A status controller is created to watch VolSync CRs. It watches the `ReplicationSource` and`ReplicationDestination` objects and updates VolumeSnapShot CR events. 
-
-*Note: Potential feature addition to Velero: A status watch controller for DataMover CRs. This can be used to update Velero Backup/Restore events with the DataMover CR results*
-
 Data mover controller will clean up all controller-created resources after the process is complete.
 
+## Alternate Design ideas
 
 ### Support for multiple data mover plugins
 `DataMoverClass` spec will support the following field,
     `selector: <tagname>`
 PVC must be labelled with the `<tagname>`, to be moved by the specific `DataMoverClass`. User/Admin of the cluster must label the PVCs with the required `<tagname>` and map it to a `DataMoverClass`. If the PVCs are not labelled, it will be moved by the default datamover.
 
 #### Alternate options
-PVCs can be annotated with the `DataMoverClass`, and when a backup is created, the controller will look at the DataMoverClass and add it to the `DataMoverBackup` CR. 
-
- 
----
+PVCs can be annotated with the `DataMoverClass`, and when a backup is created, the controller will look at the DataMoverClass and add it to the `VolumeSnapshotBackup` CR. 
```

Comment:
Are we still going to do this?

Reply:
Nope. That's I just kept them under Alternate design subheading.